security_monkey/alerters/custom_alerter.py

```diff
@@ -14,32 +14,58 @@
 """
 .. module: security_monkey.alerters.custom_alerter
     :platform: Unix
-
-.. version:: $$VERSION$$
-.. moduleauthor:: Bridgewater OSS <opensource@bwater.com>
-
-
+.. version:: SlackBot Integration Alerter w/ Security_Monkey
 """
 from security_monkey import app
+from slack_post import postMessage
+import os
 
 alerter_registry = []
 
-
 class AlerterType(type):
-
-    def __init__(cls, name, bases, attrs):
+    def __init__(self, cls, name, bases, attrs):
         if getattr(cls, "report_auditor_changes", None) and getattr(cls, "report_watcher_changes", None):
             app.logger.debug("Registering alerter %s", cls.__name__)
             alerter_registry.append(cls)
 
-
 def report_auditor_changes(auditor):
-    for alerter_class in alerter_registry:
-        alerter = alerter_class()
-        alerter.report_auditor_changes(auditor)
+       for item in auditor.items:
+            for issue in item.confirmed_new_issues:
```

Comment:
Please remove the `print()` statements. I'm not sure if it's necessary to log these details, as I *think* they are already logged elsewhere.

In this case, the logic for posting to slack is much simpler -- if something is seen, then just post it to slack. See the first comment about the `some_function_that_does_the_logic_for_posting_stuff_to_slack()`, which is where that logic should be placed.